src/main/resources/db/migration/common/V1_403__automated_task_users.sql

```diff
@@ -0,0 +1,94 @@
+INSERT INTO user_account
+VALUES (-6, NULL, 'System user for ProcessDailyList Job', NULL,
+        'systemProcessDailyListAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true,
+        true, 'systemProcessDailyListAutomatedTask'),
+       (-7, NULL, '', NULL,
+        'systemCloseOldUnfinishedTranscriptionsAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true,
```

Comment:
Any specific reason to prefix them all with `system`? Seems overkill as they are set as "system" users as well...

Reply:
This was to follow the pattern with the other system users already in the database.